src/mod/formats/mod_vlc/mod_vlc.c

```diff
@@ -1096,7 +1096,7 @@ static switch_status_t vlc_file_read(switch_file_handle_t *handle, void *data, s
 		switch_thread_cond_wait(context->cond, context->cond_mutex);
 		status = libvlc_media_get_state(context->m);
 	}
-	switch_mutex_lock(context->cond_mutex);
+	switch_mutex_unlock(context->cond_mutex);
```

Comment:
LGTM. But please put a new line before the unlock.

Reply:
resolved